forge-gui/res/cardsfolder/upcoming/cosmium_confluence.txt

```diff
@@ -0,0 +1,12 @@
+Name:Cosmium Confluence
+ManaCost:4 G
+Types:Sorcery
+A:SP$ Charm | Choices$ DBChangeZone,DBPutCounter,DBDestroy | CharmNum$ 3 | CanRepeatModes$ True
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Library | Destination$ Battlefield | Tapped$ True | ChangeType$ Card.Cave | ChangeNum$ 1 | SpellDescription$ Search your library for a Cave card, put it onto the battlefield tapped, then shuffle.
+SVar:DBPutCounter:DB$ PutCounter | Choices$ Cave.YouCtrl | CounterType$ P1P1 | RememberCards$ True | CounterNum$ 3 | SubAbility$ DBAnimate | SpellDescription$ Put three +1/+1 counters on a Cave you control. It becomes a 0/0 Elemental creature with haste. It's still a land.
+SVar:DBAnimate:DB$ Animate | Defined$ Remembered | Power$ 0 | Toughness$ 0 | Types$ Creature,Elemental | Keywords$ Haste | Duration$ Permanent | SubAbility$ DBCleanup
```

Comment:
it needs to work even vs. _Solemnity_

Reply:
So I guess an extra choosecard line or is there a way to remember the chosen card on the putcounter line ?

Reply:
probably

Reply:
you need to take note that ChooseCard is optional by default